src/common/Env.cpp

```diff
@@ -57,6 +57,8 @@ Env::Env()
     , twitchEmoteSetResolverUrl(readStringEnv(
           "CHATTERINO2_TWITCH_EMOTE_SET_RESOLVER_URL",
           "https://braize.pajlada.com/chatterino/twitchemotes/set/%1/"))
+    , imageUploaderUrl(readStringEnv("CHATTERINO2_IMAGE_PASTE_SITE_URL",
```

Comment:
Change the environment variable name to be `CHATTERINO2_IMAGE_UPLOADER_URL`

this is to make it fit with the actually variable name here